### ISSUE TITLE
Enharmonies

### ISSUE THREAD

**Issue by allorens** [DCMLab/standards#25]
Are annotators encouraged to consider enharmonies when a chord seems out of context in the way it is written?
See the example below (in Bb major): b. 50 is, as written, a minor 4th degree with augmented fifth yet it functions as the dominant of the following chord (Eb=D# and Gb=F#). It's a 1733 piece, so no augmented 5ths in general (and in other pieces by the same composer, Caldara). What would you tell annotators?

![image](https://user-images.githubusercontent.com/61057980/86763042-576a0b80-c047-11ea-846b-8c5d83c8cc80.png)

**Comment by johentsch** [comment]
This is not mentioned in the guidelines but I think the way annotators go at it so far is to care less about enharmonic spelling than about the function that the chord conveys in its context. In this particular case, I have trouble considering the chord over Eb as a B major chord because I cannot hear it as anything else than a chord over `4` (rather than over `#3`) and because I deem V/#iv an unplausible chord (#iv naturally holding a diminished chord). Especially taking into account the melody, for me it would be a clear case for `iv` or `iv(b6) iv`.

**Comment by allorens** [comment]
Look at the following bar: the e minor harmony is the `#iv` with respect to the local key `Bb` - therefore no diminished chord. Thus my enharmonisation of Eb-Gb-B as a B major chord (D#-F#-B) functioning as the dominant of e minor (also note the A in the last note of the vocal part in that bar: it can be see as a passing 7th of that `V65/#iv`).

**Comment by johentsch** [comment]
Maybe I am blind but I don't see an E minor chord; otherwise I would totally agree.

**Comment by allorens** [comment]
You're right, I'm the blind/imaginative one: I was extending the natural from the previous bar (which one has to do very frequently in these sources, but not here). In that case, I'd enharmonise the Eb-Gb-B as Eb-Gb-Cb (`.bI`)
